docker/Dockerfile.exp

```diff
@@ -4,12 +4,15 @@ FROM esmvalgroup/esmvalcore:latest
 
 COPY ./environment.yml /src/ESMValTool/environment.yml
 WORKDIR /src/ESMValTool
-RUN conda update -n base -y conda && conda install -y -c conda-forge mamba && mamba env create --name esmvaltool --file environment.yml && conda clean --all -y
+RUN conda update -n base -y conda && conda install -y -n base -c conda-forge mamba
```

Comment:
What happened to the `mamba env create`? Is it not needed anymore?

Reply:
I'll redo the env creation at the top of it all